addons/web/static/src/views/list/list_renderer.scss

```diff
@@ -161,6 +161,9 @@
         tbody > tr.o_group_header {
             > th,
             > td {
+                background-color: $o-gray-100;
+                background-clip: padding-box;
```

Comment:
shoudn't we put all the added rules on ´.o_group_header´?
I know that you use ´background-clip´ because of Firefox but it will be not useful anymore if you put these rules on the parent (´.o_group_header´).
font-weight doesn't work on ´o_group_name´ because of ´fw-bold´.